run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+config_file="config/settings.yaml"
+stop_flag="stop.flag"
+
+cleanup() {
+    echo " Stopping..."
+    rm -f "$stop_flag"
+    exit 0
+}
+
+if [ ! -f "$config_file" ]; then
+    echo "$config_file not found"
+    exit 1
+fi
+
+run_timeout=$(sed -n '/run_timeout/s/.*: //p' "$config_file")
+
+trap 'cleanup' INT TERM
+
+while true
+do
+    if [ -f "$stop_flag" ]; then
+        cleanup
+    fi
+
+    if ! python3 src/main.py; then
+        echo "Error in src/main.py..."
+    fi
+
+    sleep "$run_timeout"
+done
```

Comment:
what is this?

Reply:
The script will start `src/main.py` every hour to avoiding continuous requests. I was thinking that doing this all the time might be resource consuming

Reply:
we could do this using cronjob, check what is cron and how it works
here is an example of cronjob https://github.com/azerothcore/telegram-automated-db-backup#optional-run-it-daily-using-crontab to run a script everyday

linux and mac supports it easily